BaseStation/VideoStreamer/VideoWindow.xaml.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
```

Comment:
This file is duplicated with `BaseStation/MainWindow/VideoStreamer/VideoWindow.xaml.cs`